### PR TITLE
added HistoricalEvent and Tag models

### PR DESCRIPTION
**Pull request by snugthejoiner**
Got sidelined by Postgres issues on my machine. But two models are built. I'll start looking at seeds/factories/specs next.

### CODE REVIEW

File: spec/factories/historical_events.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :historical_event do
+    description "MyString"
+starting "2015-08-05"
+ending "2015-08-05"
+premilestone_status "MyString"
+ongoing_status "MyString"
+  end
+
```
Comment (Brit200313) [inline spec/factories/historical_events.rb]: Identing needs to be fixed here. It's hard to read and may lead to issues.

File: spec/models/historical_event_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe HistoricalEvent, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (Brit200313) [inline spec/models/historical_event_spec.rb]: Identing needs to be fixed here. It's hard to read and may lead to issues.

**Comment by Brit200313** [comment]
@snugthejoiner please fix the indenting in your factories and it will be good to merge. 

Also, Postgres issues can be the worst and I am glad you were able to sort them out. 

popkey

File: spec/models/historical_event_spec.rb (hunk 2 above)
Comment (snugthejoiner) [inline spec/models/historical_event_spec.rb]: I fixed the factories, I'm not sure what to do about this one. All of these were auto-generated. Should `pending...` be aligned with `RSpec.describe...`?

**Comment by snugthejoiner** [comment]
Pushed corrected factories

File: spec/models/historical_event_spec.rb (hunk 2 above)
Comment (Brit200313) [inline spec/models/historical_event_spec.rb]: I tagged the wrong file. All good!

Comment (snugthejoiner) [inline spec/models/historical_event_spec.rb]: Groovy. So will you be merging?

**Comment by Brit200313** [comment]
By the power of Greyskull....

popkey

**Comment by snugthejoiner** [comment]
I did not realize He-Man was so well-beloved on the Internet.

**Pull request merged**